### DISCUSSION TITLE
How to set time to live attribute using electordb

### DISCUSSION THREAD

**Discussion by slim-hmidi** [tywalch/electrodb#195] [Q&A]
Hi,
I'm creating a table and I need to enable the `TTL` attribute automatically using ElectroDB.
Is there any way to do that or should I set it manually on AWS when I create the table?
Thanks.

**Comment by tywalch** [comment]
Yes you can use the `default` property on an attribute, something like this: 
```
const ONE_DAY = 1000 * 60 * 60 * 24;

// seven days
const ITEM_LIFETIME = ONE_DAY * 7;
{
...
   attributes: {
      ttl: {
         type: 'number',
         default: () => Date.now() + ITEM_LIFETIME
      }
   }
...
```

**Reply by slim-hmidi** [reply]
@tywalch thanks for your answer. But, I mean if I add ttl attribute in an entity. The table will set the `ttl` to true?

**Reply by tywalch** [reply]
You will need to configure your table (separately) to enable TTL functionality and specify the field that will contain the TTL:

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html
